File: src/main/java/com/github/jchrist/radicleintellijplugin/services/MyProjectService.java
package com.github.jchrist.radicleintellijplugin.services;

import com.intellij.openapi.project.Project;
import com.github.jchrist.radicleintellijplugin.MyBundle;

public class MyProjectService {

    public MyProjectService(Project project) {
        System.out.println(MyBundle.message("projectService", project.getName()));
    }
}

File: src/main/kotlin/com/github/jchrist/radicleintellijplugin/services/MyProjectService.kt
package com.github.jchrist.radicleintellijplugin.services

import com.intellij.openapi.project.Project
import com.github.jchrist.radicleintellijplugin.MyBundle

class MyProjectService(project: Project) {

    init {
        println(MyBundle.message("projectService", project.name))
    }
}
